app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -0,0 +1,312 @@
+---
+id: javafx.animation
+title: Introduction to JavaFX animations
+slug: learn/javafx-animations
+type: tutorial
+group: rich-client-apps
+category: javafx
+category_order: 2
+layout: learn/tutorial.html
+subheader_select: tutorials
+main_css_id: learn
+description: "Learn to create advanced JavaFX animations"
+last_update: 2024-05-22
+---
+
+The [javafx.animation](javafxdoc:AnimationPackageSummary) package offers a simple framework for creating animations and transitions in a JavaFX application.
+It operates on the principle of [`WritableValue\<T\>`](javafxdoc:WritableValue), which is used across JavaFX. `WritableValue<T>` is an interface that wraps a value that can be read and set.
```

Comment:
When I render this, the backslashes are displayed due to this being an inline codeblock.
![image](https://github.com/java/devjava-content/assets/34687786/85e1675b-689e-4abd-85b2-4b49c7f25059)